src/store/configureStore.js

```diff
@@ -6,7 +6,7 @@ import { composeWithDevTools } from 'redux-devtools-extension';
 import rootEpic from './rootEpic';
 import rootReducer from './rootReducer';
 
-export default function configureStore() {
```

Comment:
Why is default removed?

Reply:
This was simply to have consistency between the normal configureStore and the one used in the walkthrough part of the code. They are both referenced in the `index.js` file and so at least one of them needs to be able to be renamed so we can refer to both in the same file...
```js
import { configureStore } from 'store/configureStore';
import { configureStore as configureWalkthroughStore } from 'walkthrough/store/configureStore';
```